docs/content/tendermint-and-cosmos.rst

```diff
@@ -0,0 +1,93 @@
+Tendermint & Cosmos
```

Comment:
I think Tendermint and Cosmos each deserve their own section

Reply:
Also it's not clear what this section is for. Is this a more detailed overview of Tendermint and Cosmos or an aggregator of github links. Seems like there's some overlapping with the "App development" section

Reply:
> Is this a more detailed overview of Tendermint and Cosmos or an aggregator of github links.

It's both. T&C on Github has a ton of repos, this page is meant to orient folks, so they can easily figure out which repo they want to go check out